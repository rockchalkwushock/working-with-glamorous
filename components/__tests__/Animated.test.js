import { shallow, render, mount } from 'enzyme'

import { AnimatedDiv } from '../index'

describe('AnimatedDiv Component', () => {
  let ui

  test('should render to DOM', () => {
    ui = <AnimatedDiv />
    expect(ui.type.displayName).toEqual('AnimatedDiv')
    expect(shallow(ui)).toMatchSnapshotWithGlamor(`enzyme.shallow`)
    expect(mount(ui)).toMatchSnapshotWithGlamor(`enzyme.mount`)
    expect(render(ui)).toMatchSnapshotWithGlamor(`enzyme.render`)
  })
})
