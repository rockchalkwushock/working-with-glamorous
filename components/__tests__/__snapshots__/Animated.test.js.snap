// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enzyme.mount 1`] = `
.css-837fxu,
[data-css-837fxu] {
  -webkit-animation: animation_1y5l68o 0.2s infinite ease-in-out alternate;
  animation: animation_1y5l68o 0.2s infinite ease-in-out alternate;
}

<AnimatedDiv>
  <div
    className="css-837fxu"
  />
</AnimatedDiv>
`;

exports[`enzyme.render 1`] = `
.css-837fxu,
[data-css-837fxu] {
  -webkit-animation: animation_1y5l68o 0.2s infinite ease-in-out alternate;
  animation: animation_1y5l68o 0.2s infinite ease-in-out alternate;
}

<div
  class="css-837fxu"
/>
`;

exports[`enzyme.shallow 1`] = `
.css-837fxu,
[data-css-837fxu] {
  -webkit-animation: animation_1y5l68o 0.2s infinite ease-in-out alternate;
  animation: animation_1y5l68o 0.2s infinite ease-in-out alternate;
}

<div
  className="css-837fxu"
/>
`;
